site/_posts/2017-08-29-where-have-all-the-cow-wranglers-gone.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Last Week in ManageIQ: Where Have All The Cow Wranglers Gone"
+author: allenbw
+date: 2017-08-29
+comments: true
+published: true
+tags: LWIMIQ
+---
+Hi Friends!  [Allen](https://github.com/allenbw) here, with news to share!
+Since [we last talked](http://manageiq.org/blog/2017/08/back-to-basics/) much has happened, two weeks of on goings are due for discussion.
+A few quick stats before we dig in:
+* No new repositions this time around!
+* ~ 447 PRs closed across 133 ManageIQ Repos (between 08/07 to 08/21)
+* In descending order, the top four repositories by pull request activity:
+    * [UI Classic](https://github.com/ManageIQ/manageiq-ui-classic)
+    * [ManageIQ](https://github.com/ManageIQ/manageiq)
+    * [Integration Tests](https://github.com/ManageIQ/integration_tests)
+    * [ManageIQ Docs](https://github.com/ManageIQ/manageiq_docs)
+* Downstream version updated from 4.5 to 4.6 [here](https://github.com/ManageIQ/manageiq_docs/pull/491)
```

Comment:
Nitpicking - missing period at the end of sentence 😝

Reply:
![screenshot from 2017-08-29 19-18-32](https://user-images.githubusercontent.com/2913667/29831816-fa39c0e2-8cee-11e7-9158-3079b045ecdb.png)

Reply:
should be a space! fixed it 🙇‍♀️ thanks!!